src/platform/web/ui/css/themes/element/theme.css

```diff
@@ -251,6 +251,45 @@ a.button-action {
 
 .PreSessionScreen {
     padding: 30px;
+    max-height: calc(100% - 30px); /* removing padding */
+    display: flex;
+    overflow: auto;
+    flex-direction: column;
+}
+
+@media screen and (min-width: 600px) {
+    .PreSessionScreen {
+        max-height: calc(100% - 30px - 80px); /* removing padding and margin */
```

Comment:
Just an idea, but would this be simpler if we use `box-sizing: content-box`?

Reply:
Thanks ^^'

Reply:
it doesn't work because of the margin